docs/UNIT_TEST.md

```diff
@@ -0,0 +1,33 @@
+# UNIT Tests
+
+This documentation covers everything about developing and running the UNIT tests.
+
+## Introduction to E2E Tests
+
+#### What is an E2E Test?
+Unit tests allow you to test individual units of code in isolation. 
+The purpose of unit testing is to provide developers with confidence in their code.
+
+
+#### How do we test the UI?
```

Comment:
Do we actually test the UI? I think we do not. The e2e tests do test the UI, unit tests just test the components themselves, separated.